app/components/Avatar.tsx

```diff
@@ -42,31 +34,43 @@ export const Avatar: React.FC<AvatarProps> = (props) => {
   const $imageStyle = Object.assign({}, $viewPresets[preset], imageStyle)
   const $containerStyle = [$baseContainerStyle, $styleOverride]
 
-  if (preset !== "speaker-panel") {
-    return (
-      <View style={$containerStyle}>
+  return (
+    <View style={$containerStyle}>
+      {props.sources.map((source, index) => (
         <Image
-          source={(props as SingleAvatarProps).source}
-          style={$imageStyle}
+          key={`panel-${index}-${source}`}
+          source={source}
+          style={[$imageStyle, $panelImageStyle]}
           resizeMode="contain"
         />
-      </View>
-    )
-  } else {
-    const sources = (props as PanelAvatarProps).sources
-    return (
-      <View style={$containerStyle}>
-        {sources.map((source, index) => (
-          <Image
-            key={`panel-${index}-${source}`}
-            source={source}
-            style={[$imageStyle, $panelImageStyle]}
-            resizeMode="contain"
-          />
-        ))}
-      </View>
-    )
-  }
+      ))}
+    </View>
+  )
+  // if (preset !== "speaker-panel") {
```

Comment:
We could look to use the `speaker-panel` styling if speakers is more than 1 maybe? If they still want to use that styling that is.

Reply:
updated in next push